ci/ray_ci/tester.py

```diff
@@ -416,8 +417,33 @@ def _get_high_impact_test_targets(
         if test.get_oncall() == team
     }
     changed_tests = _get_changed_tests()
+    human_specified_tests = _get_human_specified_tests()
 
-    return high_impact_tests.union(changed_tests)
+    return high_impact_tests.union(changed_tests).union(human_specified_tests)
+
+
+def _get_human_specified_tests() -> Set[str]:
+    """
+    Get all test targets that are specified by humans
+    """
+    base = os.environ.get("BUILDKITE_PULL_REQUEST_BASE_BRANCH")
+    head = os.environ.get("BUILDKITE_COMMIT")
+    if not base or not head:
+        # if not in a PR, return an empty set
+        return set()
+
+    tests = set()
+    subprocess.check_call(["git", "fetch", "origin", base], cwd=bazel_workspace_dir)
```

Comment:
I think buildkite always checks out all. do we really need to fetch from origin again here?

Reply:
let's me try